app/scripts/lib/app-start.js

```diff
@@ -601,8 +601,19 @@ define(function (require, exports, module) {
       // was used to sign up to allow the verification tab to have
       // the same capabilities as the signup tab.
       // If verifying in a separate browser, fall back to the default context.
-      const verificationInfo = this._getSameBrowserVerificationModel('context');
-      return verificationInfo.get('context');
+      const sameBrowserVerificationContext = this._getSameBrowserVerificationModel('context').get('context');
+      if (sameBrowserVerificationContext) {
+        // user is verifying in the same browser
```

Comment:
Ah yea we can just update that, as mentioned in https://github.com/mozilla/fxa-content-server/pull/4995/files#r114783595